File: inventorybot-service/src/main/kotlin/com/inventorybot/model/OrderModel.kt
package com.inventorybot.model

class OrderModel{
  var orderId:String?=null
  var appRegisteredId:String?=null
  var storeId:String?-null
  var creadtedBy:String?=null
  var createdDate :String?=null
  var products:List<ProductModel>?=null
}
class ProductModel
{
  var productId:String?=null
  var productName:String?=null
  var productPrice:String?=null

}

File: inventoryguidebot-service/inventoryguidboat-repository/Models/OrderModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace inventoryguidboat_repository.Models
{
    public class OrderModel
    {
        public string OrderId { get; set; }
        public string AppRegisteredId { get; set; }
        public string StoreId { get; set; }
        public string CreatedBy { get; set; }
        public String CreatedDate { get; set; } = System.DateTime.Now.ToString();
        public List<ProductModel> Products { get; set; }
    }
}
